setup/windows/README.md

```diff
@@ -49,6 +49,6 @@ $ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml -vv
 ```
 
 The servers should logon automatically at boot and start the Jenkins slave, but there is no automatic restart procedure for crashes (for now).
-Hence, to start Jenkins after running `ansible-playbook`, the server must be restarted or the slave must be manually started with RDP.
```

Comment:
@kunalspathak that line is outdated now, I wrote it before I added the loop in `jenkins.bat` and never removed it. You suggestion is perfect, please go ahead and update this! Thanks for noticing!